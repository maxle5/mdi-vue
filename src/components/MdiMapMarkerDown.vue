
<template>
  <svg :width="size" :height="size" viewBox="0 0 24 24">
      <path d="M12 2A7 7 0 0 0 5 9C5 14.25 12 22 12 22S19 14.25 19 9A7 7 0 0 0 12 2M7.5 10H10V5H14V10H16.5L12 14.5Z" style="fill: currentColor" />
  </svg>
</template>

<script lang="ts">
  import { defineComponent } from 'vue';

  export default defineComponent({
      name: "MdiMapMarkerDown",
      props: {
          size: { 
              type: Number, 
              default: 24 
          }
      },
  })
</script>
