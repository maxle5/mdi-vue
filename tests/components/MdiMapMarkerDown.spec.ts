
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiMapMarkerDown from "../../src/components/MdiMapMarkerDown.vue";

test("MdiMapMarkerDown snapshot", () => {
  const wrapper = mount(MdiMapMarkerDown, {});
  expect(wrapper.html()).toMatchSnapshot();
});
