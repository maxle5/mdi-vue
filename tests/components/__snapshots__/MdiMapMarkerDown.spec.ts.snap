// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiMapMarkerDown snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M12 2A7 7 0 0 0 5 9C5 14.25 12 22 12 22S19 14.25 19 9A7 7 0 0 0 12 2M7.5 10H10V5H14V10H16.5L12 14.5Z" style="fill: currentColor;"></path>
</svg>"
`;
